packages/react-devtools-shared/src/backend/agent.js

```diff
@@ -483,7 +482,12 @@ export default class Agent extends EventEmitter<{
         this._persistedSelection = null;
         this._persistedSelectionMatch = null;
         renderer.setTrackedPath(null);
-        this._throttledPersistSelection(rendererID, id);
+        // Throttle persisting the selection.
+        this._lastSelectedElementID = id;
+        this._lastSelectedRendererID = rendererID;
+        if (this._persistSelectionTimer === null) {
+          setTimeout(this._persistSelection, 1000);
+        }
```

Comment:
`this._persistSelectionTimer` is never non-null here, I am not sure if this is by design

Reply:
Looks like it was never initialized even before this change, but at least was throttled. Should probably initialize timer value here to avoid calling `sessionStorageSetItem` too many times?

Reply:
Yea it’s an oversight. Doesn’t need to be a timer but just a boolean.